### PR TITLE
CHE-550: fix projects views

### PR DESCRIPTION
**Pull request by olexii4**
**_3 Upvotes**_ Signed-off-by: Oleksii Orel oorel@codenvy.com
@ashumilova WDYT?

### CODE REVIEW

File: dashboard/src/app/workspaces/workspace-details/workspace-details.controller.js [hunk 1]
```diff
@@ -46,6 +46,8 @@ export class WorkspaceDetailsCtrl {
     } else {
       this.updateWorkspaceData();
     }
+
```
Comment (benoitf) [inline dashboard/src/app/workspaces/workspace-details/workspace-details.controller.js:49]: why do we have changes on workspaces if issue is about projects ?

**Comment by olexii4** [comment]
DONE

**Comment by ashumilova** [comment]
+1

**Comment by benoitf** [comment]
+1 good fixes

**Comment by akurinnoy** [comment]
+1

**Pull request merged**